<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SatelliteRadioControlDialog</class>
 <widget class="QDialog" name="SatelliteRadioControlDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>771</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Liberation Sans</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Satellite Radio Control</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <layout class="QGridLayout" name="gridLayout">
      <item row="7" column="0">
       <widget class="QLabel" name="satelliteModesLabel">
        <property name="text">
         <string>Satellite modes</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <layout class="QHBoxLayout" name="buttonsHorizontalLayout"/>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="satHorizontalLayout">
        <item>
         <widget class="QLabel" name="satelliteSelectlabel">
          <property name="text">
           <string>Satellite</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="satelliteSelect">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="add">
          <property name="toolTip">
           <string>Add device set control settings tab</string>
          </property>
          <property name="text">
           <string>Add device set</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="satHorizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="8" column="0">
       <widget class="QTextEdit" name="satelliteModes">
        <property name="toolTip">
         <string>Satellite modes from SatNOGS</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="tabsClosable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>satelliteSelect</tabstop>
  <tabstop>satelliteModes</tabstop>
 </tabstops>
 <resources>
  <include location="../../../sdrgui/resources/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SatelliteRadioControlDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SatelliteRadioControlDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
